docs/Adding-Software-to-Homebrew.md

```diff
@@ -2,13 +2,18 @@
 
 Are you missing your favorite software in Homebrew? Then you're the perfect person to resolve this problem.
 
-Before you start, please check the open pull requests for [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls), to make sure no one else beat you to the punch.
+If you want to add software that is either closed source, or a GUI only program you will want to follow the guide for [Casks](#casks). Otherwise follow the guide for [Formulae](#formulae).
+See also: https://docs.brew.sh/Formula-Cookbook#homebrew-terminology
```

Comment:
I might be wrong about this but I think links aren't auto-linked in the GitHub pages conversion, so maybe to be safe:

```suggestion
See also: [Homebrew Terminology](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology)
```

(or just test it and see)